docs/components/InputFile/Web.stories.tsx

```diff
@@ -51,6 +51,29 @@ const StatefulTemplate: ComponentStory<typeof InputFile> = args => {
   }
 };
 
+const MaxFilesTemplate: ComponentStory<typeof InputFile> = args => {
+  const [files, setFiles] = useState<FileUpload[]>([]);
```

Comment:
I think it would be worth adding the "manually" managing the state for the errors described here
<img width="825" alt="image" src="https://github.com/user-attachments/assets/b3e8527a-3782-4fb8-817d-bcac394c121b">

That way it is more clear as to how to manage it instead of leaving it up to developers to figure it out